.travis.yml

```diff
@@ -6,6 +6,9 @@ git:
   depth: false
 
 env:
+  global:
+  # encrypt with: travis encrypt --com --repo playframework/play-grpc  WHITESOURCE_PASSWORD=...
```

Comment:
The command I used is slightly different from that we used in other repos. My `travis` CLI is on version `1.9.1` already so you may need to update first.

`travis encrypt --com --repo playframework/play-grpc ...`

1. `--com` is an alias to `--pro` and they are required because we target `https://travis-ci.com` (not `https://travis-ci.org`.
2. I didn't use `-add` to avoid too much code changes. The `travis` CLI will read and write the file re-formatting it for you introducing multiple changes.
3. `--repo playframework/play-grpc ` is important to make sure the CLI builds the secret for the correct remote on your local working copy. I have a remote pointing to my fork and that was being picked up by default.